crates/bevy_input/src/gamepad.rs

```diff
@@ -467,7 +295,7 @@ pub enum GamepadAxisType {
 ///
 /// ## Updating
 ///
-/// The resource is updated inside of the [`gamepad_event_system`].
+/// The resource is updated inside of the gamepad event handling systems.
```

Comment:
Nit: Do we want to maintain the link to these systems still? Same for the other similar doc comments.

Reply:
If this comment exists, it should be linked.